web/webpack.config.js

```diff
@@ -14,9 +14,9 @@ module.exports = {
         "app": "./static/app/webpack.index.js",
     },
     output: {
-        path: __dirname + '/static/dist',
+        path: path.resolve(__dirname, 'static/dist'),
         filename: "[name]/webpack.bundle-[hash].js",
-        publicPath: "/en/dist/",
+        publicPath: "/",
```

Comment:
We need it as `/en/dist/` because it where bundle are available are static files.
So I think we need to mimics this `publicPath: "/en/dist/",` in `devServer`. But I cannot get it works for me.